ReactiveUI/PropertyBinding.cs

```diff
@@ -906,6 +906,11 @@ public class PropertyBinderImplementation : IPropertyBinderImplementation
             object conversionHint = null,
             IBindingTypeConverter vmToViewConverterOverride = null)
         {
+
+            if (target == null) {
+                throw new ArgumentNullException("target", "Can't bind to a property on null.  To fix this, you probably want BindTo(this, ...) isntead. ");
```

Comment:
Hmmm, I think simply throwing an `ArgumentNullException` for `target` will suffice. Trying to be more helpful here feels a little misleading/confusing.